Jenkinsfile

```diff
@@ -151,7 +151,7 @@ pipeline {
         slackSend (
           channel: "#frontend-ci-status",
           color: "warning",
-          message: "UNSTABLE: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]' (${env.RUN_DISPLAY_URL})",
+          message: "UNSTABLE: ${env.BRANCH_NAME} Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]' (${env.RUN_DISPLAY_URL})",
```

Comment:
when triggered from a PR, `BRANCH_NAME` is `PR-1234` :(
But `CHANGE_BRANCH` could work (it is empty when triggered from a branch)

Reply:
sure, let's try CHANGE_BRANCH